db/migrate/20150925022438_add_organization_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddOrganizationToUsers < ActiveRecord::Migration
+  def change
+  	add_column :users, :organization, :string, :default => "college"
```

Comment:
Hmm, after our design discussion I thought that we agreed `organization`s were to be qsets (specifically, qsets without parents). How are we sure that the string `'college'` is actually a qset in our app? I think this needs to be reworked, using a foreign relationship with qsets. Also, if `'college'` is the default, that means existing users are forced to be in that organization; is there a consideration to whether they wish to be or not? (e.g. let's think about the existing 'prod' version of AskUp and its use cases -- its current users may not want to be in the `'college'` organization.)